openmc/model/model.py

```diff
@@ -1,11 +1,32 @@
 from collections.abc import Iterable
+import operator
+import os
 from pathlib import Path
+from numbers import Integral
 import time
+import warnings
+import subprocess
+from contextlib import contextmanager
 
 import h5py
 
 import openmc
+from openmc.dummy_comm import DummyCommunicator
+from openmc.executor import _process_CLI_arguments
 from openmc.checkvalue import check_type, check_value
+from openmc.exceptions import InvalidIDError
+
+
+@contextmanager
+def _change_directory(working_dir):
+    """A context manager for executing in a provided working directory"""
+    start_dir = Path().cwd()
```

Comment:
`cwd` is actually a classmethod:
```suggestion
    start_dir = Path.cwd()
```